src/main/java/org/tinder/filters/MessagesStaticResourcesFilter.java

```diff
@@ -0,0 +1,37 @@
+package org.tinder.filters;
+
+import org.tinder.services.Services;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class MessagesStaticResourcesFilter extends RequestFilter{
+
+    public MessagesStaticResourcesFilter(Services services) {
+        super(services);
+    }
+    @Override
+    boolean accept(HttpServletRequest req, HttpServletResponse res) {
+
+        String path = req.getRequestURI().substring(req.getContextPath().length());
+        return !path.startsWith("/messages/static/");
+    }
+
+    @Override
+    void failed(HttpServletRequest req, HttpServletResponse res) {
+        String newPath = req.getRequestURI()
```

Comment:
this is redundant code

just request dispatcher from req.pathInfo

path /messages/static/css/style.css 
req.getRequestURI() =  /messages/static/css/style.css 
req.pathInfo() = /static/css/style.css (path without servlet context)

Reply:
ok